src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,13 +188,34 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resourse-dropdown [r language counts]
+  (let [organisation? (get-in r [:organization :organization/short-name (keyword language)])
+        duplicate? (> (get counts (:resid r)) 1)
+        licenses? (not-empty (:licenses r))]
+    (str (:resid r)
+      ;; when organisation is found, show organisation
```

Comment:
Oh, one more thing: I think your code is clear enough even without these comments. What do you think?